canoniCAI/utils/clf_train.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Is there a reason for this utils folder and this file? I think this file already exists under data/ right?

Reply:
remember the stuff I told you. I don't know if there was an update to jaseci infrastuture but when i ran it with the data folder it was throwing a lot of errors asking for utils folder until i provided that then it worked.